### DISCUSSION TITLE
How do you make a marker bold in a numbered list?

### DISCUSSION THREAD

**Discussion by u01jmg3** [tailwindlabs/tailwindcss#9119] [Help] [answered]
Curious how to achieve the following in TailwindCSS?

```css
ol li::marker { font-weight: bold }
```

<ol>
    <li>Item 1</li>
    <li>Item 2</li>
    <li>Item 3</li>
</ol>

**Comment by wongjn** [comment, marked as answer]
If the rule is to apply globally, you could write that verbatim in your base `@layer`:

```css
@tailwind base;
@tailwind component;
@tailwind utility;

@layer base {
  ol li:marker {
    font-weight: bold;
  }
}
```

See this [Tailwind Play example](https://play.tailwindcss.com/j1mTlUoQVv?file=css)

---

Otherwise, if it is styling for a particular `<ol><li/></ol>` element structure:
- For `ol li` portion, we can use one of these arbitrary variants:
  1. `[ol_&]` if applying on to the `li`s
  2. `[&_li]` if applying on to the `ol`
  3. `[&_ol_li]` if applying on to some parent container
- For `::marker` portion, we can use the `marker` variant
- For `font-weight: bold`, we use the `font-bold` utility

This results in:

1. ```html
   <div class="pl-5">
     <ol class="list-decimal">
       <li class="[ol_&]:marker:font-bold">Item 1</li>
       <li class="[ol_&]:marker:font-bold">Item 2</li>
       <li class="[ol_&]:marker:font-bold">Item 3</li>
     </ol>
   </div>
   ```
   [Example](https://play.tailwindcss.com/N5K4V1RhH4)  

2. ```html
   <div class="pl-5">
     <ol class="list-decimal [&_li]:marker:font-bold">
       <li>Item 1</li>
       <li>Item 2</li>
       <li>Item 3</li>
     </ol>
   </div>
   ```
   [Example](https://play.tailwindcss.com/aWVZ6jPaVP)  

3. ```html
   <div class="pl-5 [&_ol_li]:marker:font-bold">
     <ol class="list-decimal">
       <li>Item 1</li>
       <li>Item 2</li>
       <li>Item 3</li>
     </ol>
   </div>
   ```
   [Example](https://play.tailwindcss.com/6oE0yoGU3c)

**Reply by u01jmg3** [reply]
Thank you - super powerful

**Comment by justrustyspoons** [comment]
In tailwind 4 I just used marker:font-bold on the ol and it works
